exporter/sumologicexporter/README.md

```diff
@@ -0,0 +1,49 @@
+# Sumo Logic Exporter
+
+This exporter supports sending logs and metrics data to [Sumo Logic](https://www.sumologic.com/).
+
+The following configuration options are supported:
+
+- `url` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
+- `method` (optional): HTTP method, one of: `POST` or `PUT`. `POST` is used by default if unset.
+- `compress` (optional): Option to enable compression (default false)
```

Comment:
Default should be true IMO.  Collector does this and in general, we want to send data as cheap as possibly.